src/dataflow/src/sink/kafka.rs

```diff
@@ -198,6 +214,8 @@ impl KafkaSinkState {
             pending_rows: HashMap::new(),
             ready_rows: VecDeque::new(),
             send_state: SendState::Init,
+            latest_progress_ts: Timestamp::MIN,
```

Comment:
should this be set to the gate timestamp if that exists on restart?

Reply:
:+1: